storage_backend_sftp/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Storage Backend SFTP",
     "summary": "Implement SFTP Storage",
-    "version": "12.0.1.0.0",
```

Comment:
13?

Reply:
this module is not migrated yet, it comes from 12 as it is